src/components/ContentNode/Row.vue

```diff
@@ -0,0 +1,62 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <div class="row" :style="style" :class="{ 'with-columns': columns }">
+    <slot />
+  </div>
+</template>
+
+<script>
+/**
+ * A Row component that accepts an optional `columns` prop.
+ * When columns is passed, the grid will have that exact number of columns.
+ * If no `columns` provided, width is split up equally across each cell.
+ * Max number of allowed columns is 12.
+ */
+export default {
+  name: 'Row',
+  props: {
+    columns: {
+      type: Number,
+      default: null,
+      required: false,
+      validator: v => v > 0 && v <= 12,
+    },
+  },
+  computed: {
+    style: ({ columns }) => ({ '--col-count': columns || 12 }),
```

Comment:
Can we make a constant for the `12` value?

Reply:
Or if you make 12 the default instead of allowing `null`, then this conditional wouldn't be needed.

Reply:
if I make it a default, then the grid will always be 12 unless specified. But in the DocC spec, we dont require users to add a column count, they can just add items and they will fit next to each other..